css/css-fonts/font-face-local-ref.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<style>
+  html {
+    font-family: Arial;
+  }
+</style>
+<h1>Tests pass if these render identical</h1>
```

Comment:
Final but, I think. "These two lines should render identical" might be an improvement, removing the reference to the test it's in. As it is, this might be confused for a "visual test" which is a kind of manual rendering test where a human looks at the page and says pass or fail, with no reference rendering.